ssl/ech.c

```diff
@@ -76,7 +76,7 @@ static void OSSL_ECHEXT_free(OSSL_ECHEXT *e)
     return;
 }
 
-static void OSSL_ECHSTORE_entry_free(OSSL_ECHSTORE_entry *ee)
+static void OSSL_ECHSTORE_ENTRY_free(OSSL_ECHSTORE_ENTRY *ee)
```

Comment:
Still `OSSL_ECHSTORE_ENTRY_free` -> `ossl_echstore_entry_free` as this is a non-public static function.

Reply:
done